cmd/homi/setup/flags.go

```diff
@@ -150,7 +150,7 @@ var (
 
 	dockerImageIdFlag = cli.StringFlag{
 		Name:        "docker-image-id",
-		Value:       "428948643293.dkr.ecr.ap-northeast-2.amazonaws.com/gxp/client-go:latest",
+		Value:       "klaytn/klaytn",
```

Comment:
How about adding a tag as well such as `klaytn/klaytn:latest`?

Reply:
Are they different? :)

Reply:
Reflected and added url. :) Thanks.

Reply:
Nope, the default tag is `latest`. Just for more clarity and example.